src/sqlancer/MainOptions.java

```diff
@@ -126,10 +126,19 @@ public class MainOptions {
     @Parameter(names = "--use-reducer", description = "EXPERIMENTAL Attempt to reduce queries using a simple reducer")
     private boolean useReducer = false; // NOPMD
 
+    @Parameter(names = "--reduce-AST", description = "EXPERIMENTAL perform AST reduction after statement reduction")
```

Comment:
There seems to be a check against the parameter format's failing (see "General tests"), looks like parameters are not expected to contain capital letters: https://github.com/sqlancer/sqlancer/blob/97b8850ed697359d91f77950741e5bd4a0681449/test/sqlancer/TestParameterFormat.java#L20
Maybe we can use `ast` here?

Reply:
ok